packages/zoe/src/contracts/sellItems.js

```diff
@@ -71,21 +72,14 @@ export const makeContract = harden(
         );
       }
 
-      // Reallocate and complete the buyer offer.
-      const newSellerAllocation = {
-        Money: moneyAmountMaths.add(sellerAllocation.Money, providedMoney),
-        Items: itemsAmountMath.subtract(sellerAllocation.Items, wantedItems),
-      };
-
-      const newBuyerAllocation = {
-        Money: moneyAmountMaths.getEmpty(),
-        Items: itemsAmountMath.add(buyerAllocation.Items, wantedItems),
-      };
-
-      zcf.reallocate(
-        [sellerOfferHandle, buyerOfferHandle],
-        [newSellerAllocation, newBuyerAllocation],
+      // Reallocate.
+      debugger;
+      trade(
+        { offerHandle: sellerOfferHandle, gains: { Money: providedMoney } },
+        { offerHandle: buyerOfferHandle, gains: { Items: wantedItems } },
```

Comment:
Now that I understand it, I like the way that defaulting `losses` simplifies this call. In normal code, it would be correct to take advantage of the simplification without a comment, but since this is example code, meant to demonstrate how it works, I think it's worth a comment pointing out that this can omit losses because the keywords match.

Reply:
Done